packages/next/build/webpack/plugins/next-trace-entrypoints-plugin.ts

```diff
@@ -162,7 +161,15 @@ export class TraceEntryPointsPlugin implements webpack.Plugin {
               }
 
               try {
-                return compilation.inputFileSystem.readFileSync(path)
+                return await new Promise((resolve, reject) => {
+                  ;(
+                    compilation.inputFileSystem
+                      .readFile as typeof import('fs').readFile
```

Comment:
Is something wrong with the types here?

Reply:
The `inputFileSystem` from webpack isn't typed currently but is the same as the node fs